src/lmstudio/json_api.py

```diff
@@ -1089,12 +1090,22 @@ def __init__(
         super().__init__(model_key, params, on_load_progress)
 
 
+# Add new type definitions for inline parameter format
+from typing import TypeVar
+from typing_extensions import NotRequired
```

Comment:
```suggestion
```
Add `NotRequired` to the existing import from `typing_extensions` above (between the `NoReturn` and `Self` imports)